docs/assets/scss/layouts/_header.scss

```diff
@@ -181,6 +177,17 @@ button#doks-versions {
   */
 }
 
+.show--consent,
+.show--consent body{
+  height: auto !important;
+  width: 100vw !important;
+  overflow-x: hidden !important;
+}
+
+object{
+  width: 100%;
+}
```

Comment:
What was the problem/element, fix case - comment?

Reply:
Added a comment. We need this so the Interaction manager examples to not overflow the page width size